### PR TITLE
James Palmer Chitter app

### PR DESCRIPTION
**Pull request by jpalmerr**
Completed first two user stories.
Was short for time this weekend.

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
+require 'sinatra/base'
+require_relative 'lib/peep'
+
+# controller
+class Chitter < Sinatra::Base
+  get '/' do
+    @peepdeck = Peep.all
+    puts @peepdeck
```
Comment (micosmin) [inline app.rb]: I think at the end this line can be taken out of the project

File: lib/peep.rb [hunk 2]
```diff
@@ -0,0 +1,38 @@
+require 'pg'
+
+class Peep
+
+  def self.all
+
+    if ENV['ENVIRONMENT'] == 'test'
```
Comment (micosmin) [inline lib/peep.rb]: I would suggest to move the if else statement in a script that runs when the app loads

Create a file, require the pg gem, and add this if statement.
Require the file at the top of your controller
Delete the if statements

File: spec/features/home_page_spec.rb [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+# feature 'Viewing home page' do
+#   scenario 'Shows peepdeck' do
+#     visit('/')
+#     expect(page).to have_content 'My first peep!'
+#   end
+# end
+
+feature 'Viewing home page' do
+  scenario 'Shows peepdeck' do
```
Comment (micosmin) [inline spec/features/home_page_spec.rb]: You can place this scenario inside of the first feature, as it is testing the same feature

File: spec/peep_spec.rb [hunk 4]
```diff
@@ -0,0 +1,28 @@
+require 'peep'
+
+describe Peep do
+  describe '.all' do
+    it 'returns Peepdeck' do
+      PG.connect(dbname: 'chitter_app_test')
```
Comment (micosmin) [inline spec/peep_spec.rb]: I think this should you the database set in the env variable. Not sure this line is needed here anymore. Worth testing

File: app.rb (hunk 1 above)
Comment (jpalmerr) [reply app.rb]: agreed, was there for visibility but forgot to take it out

**Pull request closed without merging**